recipe/patch_yaml/ncvis-climate.yaml

```diff
@@ -0,0 +1,7 @@
+if:
+  name: ncvis-climate
+  timestamp_lt: 1739760494852  # Mon Feb 17 02:48:49 2025
+then:
+  - tighten_depends:
+      name: wxwidgets
+      max_pin: 'x.x.x'
```

Comment:
please lets only do this ">=3.2.5"

Reply:
see https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/blob/main/recipe/patch_yaml/numba.yaml for examples